lib/file-store.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const fsPromises = require('fs').promises;
+const uuidv4 = require('uuid/v4');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const config = require('../lib/config');
+
+module.exports = {
```

Comment:
Nit: I've been preferring direct assignments to `module.exports.XXX` recently - it shaves a level of indentation off of function implementations so more can fit on the screen. E.g.

```js
module.exports.upload = ...
```

vs

```js
module.exports = {
    upload: ...
}
```

Not blocking, just for your consideration.

Reply:
Avoiding indentation 👍 

I switching in all new `.js` files introduced in this PR.